src/WebJobs.Extensions.ApiHub/Common/GenerericStreamBindingProvider.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
```

Comment:
Can you fix the spelling error of this file please ("**Genereric**StreamBindingProvider")?